skinStyles/extensions/StructuredNavigation/ext.structuredNav.NavigationView.less

```diff
@@ -0,0 +1,49 @@
+/*
+ * Citizen
+ *
+ * SkinStyles for StructuredNavigation
+ * Module: ext.structuredNav.NavigationView.styles
+ * Version: REL1_39 55e2ec0
+ *
+ * Date: 2024-09-06
+*/
+
+/* ext.structuredNav.NavigationView.less */
+.mw-structurednav-navigation-container {
+	background-color: var(--color-surface-2);
+	border: 2px solid var(--border-color-base);
+	font-size: var(--font-size-base);
+	border-radius: var(--border-radius--small);
```

Comment:
Use `var( --border-radius-base )` instead, `var( --border-radius--small )` is soft-deprecated.